### DISCUSSION TITLE
App doesn't remember scroll position in files sidebar

### DISCUSSION THREAD

**Discussion by PatAbah** [Acode-Foundation/Acode#700] [Ideas]
I noticed that the current update can remember and reopen folders from the file explorer sidebar which is cool. However, it doesn't seem to remember the scroll position in the side bar. Some users like myself work with folders with many subfolders and files. The sidebar remembering the opened folders is great, and it will be awesome if it can remember scroll position as well.

I understand that reopening all the folders, especially on FTP can take sometime and might lead to an annoying interruption if user is already scrolling when the folders finally load and auto-scrolled. To prevent this, you can make it such a way the scrolling before all last opened folders are reopened will cancel the auto-scroll function. However, if the user doesn't scroll the sidebar before everything loads, then the sidebar is automatically scrolled to last position. 

**P.S**
Closing the sidebar, and reopening it also currently resets the scroll position.
To reproduce:
1. Open a folder with many files
2. Scroll to the bottom
3. Either select a file for editing/or just close the sidebar
4. Reopen the sidebar

**Result**: 
Scrollbar will be back at the top.

**Comment by deadlyjack** [comment]
This is known issue. I didn't have enough time to fix it. Bjt it will fixed in upcoming updates. Thank you for reporting it.
